src/components/swap/AdvancedSwapDetails.tsx

```diff
@@ -203,9 +211,11 @@ function TokenTaxLineItem({ trade, type }: { trade: ClassicTrade; type: 'input'
           </>
         }
       >
-        <ThemedText.BodySmall color="textSecondary">{`${currency.symbol} fee`}</ThemedText.BodySmall>
+        <ThemedText.BodySmall color="neutral2">{`${currency.symbol} fee`}</ThemedText.BodySmall>
```

Comment:
this should also have a syncing/loading skeleton. when switching FOT tokens, the old one is showing while loading

Reply:
![loading fee](https://github.com/Uniswap/interface/assets/11512321/59c60a8f-a946-4a19-9507-46417e7904b0)

Reply:
it also shows when switching to non FOT
![loadingFeeNon](https://github.com/Uniswap/interface/assets/11512321/5d21a8f8-4ed4-4fa5-af96-f4acb59ce87f)

Reply:
I did try this, and it looked visually unbalanced w/ the rest of the rows in loading state. I'm not sure how big of an issue this is, since there's is still an overall indication that that row is about to update.

@bryankuuu @callil thoughts here?

Reply:
i'll defer to design but I think it's confusing especially going from a FOT to normal and seeing a fee loading so I would strongly recommend a loader or just hiding the row until a fee has loaded

Reply:
hiding row makes a lot of sense, just added that ty